src/runtime/helper.ts

```diff
@@ -0,0 +1,44 @@
+// prettier-ignore
```

Comment:
This looks good! I would just rename it to `helpers.ts`. But besides that, nice!